src/components/ReposList/ReposList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { Table } from 'react-bootstrap'
+import RepoItem from '../RepoItem/'
+import { orderBy } from 'lodash'
+
+const ReposList = () => {
+  const repositories  = useSelector(state => state.repositories.data)
+  
+  return(
+    <Table>
+      <thead>
+        <tr>
+          <th>Repository name</th>
+          <th>Stars</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        {orderBy(Object.values(repositories), ['stargazers_count'], ['desc']).map(repo =>
```

Comment:
It's better to do sorting in selector with memoization, because each render triggers evaluation of `orderBy` which might cause performance issues